.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
```

Comment:
You need to remove these. They were added by a merge conflict and should be removed. Same goes for the `=======` and `>>>>>>> master` below.

Reply:
Thanks g